### PR TITLE
Orca slurm

### PR DESCRIPTION
**Pull request by valtandor**
I added the SLURM_CPU_BIND environment variable (set to "none") as this is required for ORCA jobs scheduled with Slurm. Also, I took out the ability to vary the OpenMPI version as binary ORCA builds are compiled against specific versions of MPI and this is known from the download.

### CODE REVIEW

**Comment by valtandor** [comment]
Yes

**Comment by boegel** [comment]
Jenkins: ok to test

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 3 out of 4 (4 easyconfigs in this PR)
Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/c50a731ff5c424042d52 for a full test report.

File: easybuild/easyconfigs/o/ORCA/ORCA-3_0_0-linux_x86-64.eb [hunk 1]
```diff
@@ -1,8 +1,7 @@
 easyblock = "PackedBinary"
 
 name = "ORCA"
-openmpiversion = '1.6.5'
-version = '3_0_0-linux_x86-64_openmpi_%s' % ''.join(openmpiversion.split('.'))
+version = '3_0_0-linux_x86-64'
```
Comment (boegel) [inline easybuild/easyconfigs/o/ORCA/ORCA-3_0_0-linux_x86-64.eb]: the `openmpi` bit is part of the source tarball, so it must stay there

**Comment by boegel** [comment]
@valtandor: needs a sync with `develop` due to #2500

**Comment by valtandor** [comment]
@boegel: Resynced. Also, the Open MPI version numbers in the ORCA 3.0.0 file have been reinstated.

**Comment by boegel** [comment]
@valtandor:

```
AssertionError: Filename '/data/jenkins/workspace/easybuild-easyconfigs-pr-builder@3/easybuild/easyconfigs/o/ORCA/ORCA-3_0_0-linux_x86-64.eb' of parsed easyconfig matches expected filename 'ORCA-3_0_0-linux_x86-64_openmpi_165.eb'
```

**Comment by boegel** [comment]
@valtandor needs a sync with `develop`

**Comment by valtandor** [comment]
@boegel should be up to date now.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in this PR)
gligar01.gligar.os - Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/012d321f536790d5155e5440b1ccaeca for a full test report.

**Comment by boegel** [comment]
Going in, thanks @valtandor!

**Pull request merged**